DigitalLearningSolutions.Web/Controllers/LearningPortalController/Current.cs

```diff
@@ -162,23 +161,6 @@ public IActionResult RequestUnlock(int progressId)
             return View("Current/UnlockCurrentCourse");
         }
 
-        [ServiceFilter(typeof(VerifyDelegateCanAccessActionPlanResource))]

```

Comment:
We don't seem to still have this ServiceFilter in the new controller, do we need this?

Reply:
We don't need this in the new controller as we have shifted to using ResourceReferenceId instead of LearningLogItemId. As any delegate can launch a resource, we don't need a service filter, and instead check if there are any learning log items with the same resource and update those.